en/docs/learn/configuring-email-otp.md

```diff
@@ -583,40 +583,24 @@ Follow the steps given below to update the user's email address.
 
 Follow the steps below to map the user claims:
 
-!!! note
-    
-    For more information about claims, see [Adding Claim
-    Mapping](../../learn/adding-claim-mapping).
-    
-
-1.  Click **Add** under **Main \> Identity \> Claims**.  
-    ![adding-claims](../assets/img/tutorials/adding-claims.png) 
+1.  Click **List** under **Main \> Identity \> Claims**.  
+    ![listing-claims](../assets/img/tutorials/listing-claims.png) 
      
-    1.  Click **Add Local Claim**.  
-        ![add-local-claim](../assets/img/tutorials/add-local-claim.png)
+    1.  Click **http://wso2.org/claims**.  
+        ![available-claim-dialects](../assets/img/tutorials/available-claim-dialects.png)
         
-    2.  Select the **Dialect** from the drop down provided and enter the
-        required information.
+    2.  Click **Edit** in **Disable EmailOTP**.
+        ![disable-emailotp-claim](../assets/img/tutorials/disable-emailotp-claim.png)
         
-    3.  Add the following:
-
-        1.  **Claim URI:**
-            `                             http://wso2.org/claims/identity/emailotp_disabled                           `
-        2.  **Display Name** :
-            `              DisableEmailOTP             `
-        3.  **Description:**
-            `              DisableEmailOTP             `
-        4.  **Mapped Attribute (s):** `              title             `
-        5.  **Supported by Default:** checked
-
-        ![filling-claim-fields](../assets/img/tutorials/filling-claim-fields.png)
+    3.  Click **Suport by Default** and **Update**.
+        ![emailotp-support-by-default](../assets/img/tutorials/emailotp-support-by-default.png)
 
     4.  Click **Add**.
 
         To disable this claim for the admin user, navigate to **Users
         and Roles \> List** and click **Users.** Click on the **User
-        Profile** link corresponding to admin account and then click
-        **Disable EmailOTP.** This will disable the second factor
+        Profile** link corresponding to admin account and then write 
+        **false** in **Disable EmailOTP.** This will disable the second factor
         authentication for the admin user.
```

Comment:
...and enter "false" as the value for the **Disable EmailOTP** field.

Reply:
addressed in dde107f and 2532c49